src/coreclr/unwinder/riscv64/unwinder.cpp

```diff
@@ -647,6 +647,7 @@ Return Value:
     ULONG UnwindIndex;
     ULONG UnwindWords;
 
+    assert(!"---------------RISCV64 ShouldNotReachHere");
```

Comment:
Delete this method instead?

Reply:
Oops. I want to put this into `RtlpUnwindCustom` not `RtlpUnwindFunctionFull`.
I will update `unwinder.cpp` and then I will request your review again.
Thank you.